config/templates/stack-template.json

```diff
@@ -89,14 +88,22 @@
       },
       "DependsOn" : ["AutoScaleController"]
     },
+    {{end}}
     "AutoScaleController": {
       "Type": "AWS::AutoScaling::AutoScalingGroup",
       "Properties": {
         "AvailabilityZones": [
```

Comment:
`AvailabilityZones` param of autoscaling group resource can be omitted all together, ASG uses zones from `VPCZoneIdentifier` which we always specify